app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,80 @@
+          <ul class="dropdown-menu">
+            <li><%= navigation_link( t('layouts.navigation.calendar'), calendar_path, :title => t('layouts.navigation.calendar_title')) %></li>
+            <li><%= navigation_link( t('layouts.navigation.completed_tasks'), done_overview_path, {:accesskey=>"d", :title=>t('layouts.navigation.completed_tasks_title')} ) %></li>
+            <li><%= navigation_link( t('layouts.navigation.feeds'), feeds_path, :title => t('layouts.navigation.feeds_title')) %></li>
+            <li><%= navigation_link( t('layouts.navigation.stats'), stats_path, :title => t('layouts.navigation.stats_title')) %></li>
+            <li><hr/></li>
+            <li id="menu_view_toggle_contexts"><%= link_to(t('layouts.toggle_contexts'), "#", {:title => t('layouts.toggle_contexts_title'), :id => "toggle-contexts-nav"}) %></li>
+            <li><%= link_to(t('layouts.toggle_notes'), "#", {:accesskey => "S", :title => t('layouts.toggle_notes_title'), :id => "toggle-notes-nav"}) %></li>
+            <%= group_view_by_menu_entry %>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+            <%= t('layouts.navigation.admin') %> <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu">
+            <li><%= navigation_link( t('layouts.navigation.preferences'), preferences_path, {:accesskey => "u", :title => t('layouts.navigation.preferences_title')} ) %></li>
+            <li><%= navigation_link( t('layouts.navigation.export'), data_path, {:accesskey => "e", :title => t('layouts.navigation.export_title')} ) %></li>
+            <li><%= navigation_link( t('layouts.navigation.import'), import_data_path, {:accesskey => "i", :title => t('layouts.navigation.import_title')} ) %></li>
+            <% if current_user.is_admin? -%>
+            <li><%= navigation_link(t('layouts.navigation.manage_users'), users_path, {:accesskey => "a", :title => t('layouts.navigation.manage_users_title')} ) %></li>
+            <% end -%>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+            <%= t('layouts.navigation.help') %> <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu">
+            <li><%= link_to t('layouts.navigation.integrations_'), integrations_path %></li>
+            <li><%= link_to t('layouts.navigation.api_docs'), rest_api_docs_path %></li>
+          </ul>
+        </li>
+        <li><%= navigation_link(icon('search'), search_path, :title => t('layouts.navigation.search')) %></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li id="minilinks">
+          <%= link_to("#{t('common.logout')} (#{current_user.display_name}) &raquo;".html_safe, logout_path, :class => 'navbar-link') %>
```

Comment:
Is the `.navbar-link` class needed? I don't see that it actually applies any styles.